miio/miioprotocol.py

```diff
@@ -90,18 +90,22 @@ def send_handshake(self, *, retry_count=3) -> Message:
         return m
 
     @staticmethod
-    def discover(addr: str = None, timeout: int = 5) -> Any:
+    def discover(addr: str = None, is_broadcast: bool = True, timeout: int = 5) -> Any:
```

Comment:
```suggestion
    def discover(addr: str = None, timeout: int = 5, *, is_broadcast: bool = True) -> Any:
```
Let's move the `is_broadcast` to be the last parameter, and make it kwarg-only for clarity.